pom.xml

```diff
@@ -203,8 +203,9 @@
                     <version>${maven.release.plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <preparationGoals>clean verify install</preparationGoals>
+                        <preparationGoals>clean install</preparationGoals>
                         <pushChanges>false</pushChanges>
+                        <arguments>-Prelease ${arguments}</arguments>
```

Comment:
We were only including the Javadoc for apis in the DDF distribution. I don't believe we have APIs for Alliance yet. Curious if adding Javadoc adds value it definitely adds size to the distributions. Not sure if they ever get used.